### ISSUE TITLE
NTR: Liver biopsy: G6Pase activity low (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9089]
New term request Liver biopsy: G6Pase activity low (segal_060420141233): Glucose-6-phosphatase; hepatic biopsy

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000200
- label: Reduced hepatic glucose-6-phosphatase activity
- parents: [Abnormal liver enzyme activity or concentration (HP:0034685)](https://hpo.jax.org/app/browse/term/HP:0034685)
- pmid: PMID:7573034
- def: Activity of glucose-6-phosphatase (EC 3.1.3.9) in the liver below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9089
- synonyms:
